tests/unit/testsuite/Bolt/Boltpay/Model/ObserverTest.php

```diff
@@ -207,4 +215,56 @@ private function _createGuestCheckout($productId, $quantity)
 
         return $cart;
     }
+
+    /**
+     * Update Feature Switches if necessary
+     *
+     * event: controller_front_init_before
+     */
+    public function updateFeatureSwitches()
+    {
+        if (Bolt_Boltpay_Model_FeatureSwitch::$shouldUpdateFeatureSwitches) {
+            Mage::getSingleton("boltpay/featureSwitch")->updateFeatureSwitches();
+        }
+    }
+
+    private function updateFeatureSwitches_setup()
```

Comment:
Let's change the name to updateFeatureSwitchesSetup

Reply:
Done